velox/vector/BaseVector.cpp

```diff
@@ -66,26 +66,31 @@ BaseVector::BaseVector(
 
 void BaseVector::ensureNullsCapacity(vector_size_t size, bool setNotNull) {
   auto fill = setNotNull ? bits::kNotNull : bits::kNull;
+  // Ensure the size of nulls_ is always at least as large as length_.
+  auto sizeToAllocate = std::max(size, length_);
```

Comment:
Just `size = std::max(size, length_)`?

Reply:
Let me rename the argument (I try not to modify argument values unless they're intended to be modified).